frontend/demo/component/timepicker/react/time-picker-validation.tsx

```diff
@@ -15,18 +14,23 @@ function Example() {
       label="Appointment time"
       helper-text="Open 8:00-16:00"
       value={currentValue.value}
+      required
       min="08:00"
       max="16:00"
       step={60 * 30}
       errorMessage={errorMessage.value}
       onValueChanged={(event) => {
         currentValue.value = event.detail.value;
       }}
-      onChange={(event: TimePickerChangeEvent) => {
-        const { min, max, value } = event.target;
-        if (value < min) {
+      onValidated={(event) => {
+        const field = event.target as TimePickerElement;
+        if (!field.value && (field.inputElement as HTMLInputElement).value) {
+          errorMessage.value = 'Invalid time format';
+        } else if (!field.value) {
+          errorMessage.value = 'Field is required';
```

Comment:
```suggestion
```suggestion
        if (!field.value) {
          const notEmpty = (field.inputElement as HTMLInputElement).value;
          errorMessage.value = notEmpty ? 'Invalid time format' : 'Field is required';
```